src/tests/Dashboard/container/SourceSearch.test.jsx

```diff
@@ -17,7 +18,7 @@ describe('Dashboard Component', () => {
       clearSources: jest.fn(),
       hasMore: false,
     };
-    const wrapper = mount(<SourceSearch {...props} />);
+    const wrapper = shallow(<MemoryRouter><SourceSearch {...props} /></MemoryRouter>);
```

Comment:
Why did you have to change this file, I thought you were to work on dictionaries only

Reply:
I made changes to the SideNavigation.jsx file to use the Link element hence I had to wrap the component being tested in the MemoryRouter

Reply:
Yah and that should be another ticket I guess because I had also changed anchors to Links and someone commented on that, or maybe change only yours to avoid many irrelevant changes as per this ticket

Reply:
This would leave all the other test files failing as a result of rendering the Link out of the MemoryRouter